submissions/html/assignment1/2203031080042/aq1.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quotes</title>
+</head>
+<body>
+    <h1>Best Quotes According to Me</h1>
+
+        <blockquote cite="https://www.imdb.com/title/tt0944947/quotes">
+            <p>"Never forget what you are, the rest of the world will not. Wear it like armor and it can never be used to hurt you."</p>
+            <footer>— <cite>Tyrion Lannister</cite></footer>
+        </blockquote>
+        <p>This quote emphasizes <q cite="https://chatgpt.com/">embracing your true self with confidence, turning potential vulnerabilities into strengths that cannot be exploited by others.</q>.</p>
+<hr>
+        <blockquote cite="https://www.imdb.com/title/tt2306299/quotes">
+            <p>"Power is always dangerous. It attracts the worst and corrupts the best."</p>
+            <footer>— <cite>Ragnar Lothbrok</cite></footer>
+        </blockquote>
+        <p>As the saying goes, <q cite="https://chatgpt.com/">the dual nature of power, suggesting it tempts those with ill intent and can morally compromise even those with noble intentions.</q>.</p>
+<hr>
+        <blockquote cite="https://www.imdb.com/title/tt5180504/quotes">

```

Comment:
Please redo the PR. The current version affects the file path for everyone. Pull the changes from upstream and Redo the PR